Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -83,18 +84,25 @@ public extension MoyaProvider {
             switch stubBehavior {
             case .never:
                 switch target.task {
```

Comment:
Not sure what you mean. There is already a `sendRequest` function, so I'm a little confused. Also I'm not sure why the we should pull only the `.never` stub behavior switch statement out into a function named `sendRequest`, I don't really understand how that's the same thing. Did you mean the entire `performNetworking` closure? I could extract that into a method named `performNetworking` to make the function shorter if that's what you mean.